man/update.dev.pkg.Rd

```diff
@@ -30,7 +30,9 @@
   NULL.
 }
 \examples{
-  # data.table::update.dev.pkg()
+\dontrun{
+  data.table::update.dev.pkg()
```

Comment:
This adds a risk of some test jobs running with run-dontrun. IMO it is better to completely exclude cl from examples, either by comment or if(FALSE) ...
I don't see a valid use case for running this example. Yet we should still have it there because the usage section is turned into update method for dev.pkg object, which is clearly wrong: https://github.com/Rdatatable/data.table/issues/5304#issuecomment-1013101379

Reply:
Running dontrun seems odd to me but ok. I like that code in dontrun is still parsed and checked which isn't the case for comments. So that leads us to if(FALSE) then. Will do.
Regarding the usage section, our .Rd contains `\method{update}{dev.pkg}` which tells it it's a method. Can't we drop the `\method` part there, and remove the `\alias{update}` at the top too?

Reply:
If it looks like a method of update but isn't actually, then that's the sort of thing that is easier to believe leads to a hang somehow.

Reply:
there is `_R_CHECK_DONTTEST_EXAMPLES_` but I don't see anything like that for `\dontrun`